scripts/release.sh

```diff
@@ -60,8 +60,18 @@ if [[ $(git status -uno --porcelain) ]]; then
   Die 'There are uncommitted changes.'
 fi
 
-# Make sure client is up to date with the latest changes.
-git checkout master
+if [[ "${SUFFIX}" = "lts" && "${VERSION}" != *.*.0 ]]; then
+  # LTS patch release is based on N.0.0-lts branch, where N is the major release number.
+  BASE_VERSION=$(echo $VERSION | sed 's/\([0-9][0-9]*\.[0-9][0-9]*\)\.[0-9][0-9]*/\1.0/')
+  # For example, LTS BOM patch release 5.0.3 would create "5.0.3-lts" branch based on the base
+  # branch "5.0.0-lts". For the details of a patch release, see boms/cloud-lts-bom/RELEASING.md.
```

Comment:
According to the code, for more clarity, I'd say
```suggestion
  # For example, LTS BOM patch release 5.1.3 would create "5.1.3-lts" branch based on the base
  # branch "5.1.0-lts". For the details of a patch release, see boms/cloud-lts-bom/RELEASING.md.
```
Or, is this really the intention?

Reply:
This BOM doesn't touch minor version. The minor version is always zero. I added clarification in the comment.

Reply:
I see. But when you do `./release.sh lts 5.1.3`, it will take this "if" branch, and `BASE_VERSION` will be `5.1.0` (according to the `sed` expression`, and then `BASE_BRANCH=5.1.0-lts`.

Or, is it that the user should never give an input like `./release.sh lts 5.1.3`? Should it be guarded?

Reply:
Now the strategy is updated. In this example, the BASE_BRANCH will be `5.1.x-lts`.